vpplink/types/cnat.go

```diff
@@ -64,12 +64,15 @@ func (t *CnatEndpointTuple) String() string {
 	)
 }
 
+func (e *CnatTranslateEntry) Key() string {
```

Comment:
I think it's arguably also VPP specific, as it is a primary key for cnat translations in VPP.
Actually it would maybe be good to provide a key for every vpplink objects 
(for control planes wanting to store a map of VPP's state)